internal/docs/exported_fields.go

```diff
@@ -5,19 +5,27 @@
 package docs
 
 import (
+	"errors"
 	"fmt"
 	"sort"
 	"strings"
 
+	"github.com/Masterminds/semver/v3"
+
 	"github.com/elastic/elastic-package/internal/fields"
+	"github.com/elastic/elastic-package/internal/packages"
 )
 
+var semver3_2_3 = semver.MustParse("3.2.3")
```

Comment:
Would it be possible to modify a test package so it uses the new code?

Reply:
Did this in the nginx package, also build it locally and seems to work, although I'm getting this:
![image](https://github.com/user-attachments/assets/ec050128-e159-4afa-a72f-5d6c802c908c)

This comes from the package-spec repo I guess, as 3.3.1 is not defined there?

Reply:
Changes to 3.4.0